sympy/physics/vector/vector.py

```diff
@@ -694,6 +694,8 @@ def applyfunc(self, f):
         for v in self.args:
             d[v[1]] = v[0].applyfunc(f)
         return Vector(d)
+    def free_symbols(self, reference_frame):
```

Comment:
need space above

Reply:
needs a docstring

Reply:
`free_symbols` is a attribute on typical SymPy objects, not a function. Make use of the `@property` decorator to turn this method into a property.